docs/index.adoc

```diff
@@ -3138,6 +3138,60 @@ usage help shows the wrong default value
 ----
 ====
 
+==== Default Values in Unreferenced Argument Groups
+When an argument group is defined with default values in the annotated fields, but during usage does not reference any of the arguments within the group, picocli will instantiate those objects to their declared values.  However, picocli will instantiate the other objects to their default annotated values if a single argument within the argument group is defined.
```

Comment:
It is tricky to give a short and clear description of the problem we are trying to solve here... How about something like this?

> If none of the options in an argument group is specified on the command line, picocli will not instantiate the user object for that group. So how does the application access default values for the group options? Essentially, the application needs to do some extra work for groups, that is not necessary for options outside of argument groups.

Then maybe follow this by a recommended usage (see also below).